libopae/plugins/xfpga/properties.c

```diff
@@ -224,11 +222,18 @@ fpga_result __FPGA_API__ xfpga_fpgaUpdateProperties(fpga_token token,
 	SET_FIELD_VALID(&_iprop, FPGA_PROPERTY_FUNCTION);
 
 	// only set socket id if we have it on sysfs
-	result = sysfs_get_socket_id(_token->device_instance,
-				     _token->subdev_instance,
-				     &_iprop.socket_id);
-	if (0 == result)
+	result = sysfs_get_fme_path(_token->device_instance,
+	_token->subdev_instance, spath);
```

Comment:
indentation is off

Reply:
fixed
Pointer to fpga region used in function sysfs_get_fme_path() 
Port & FME objects parent is fpaga region